src/utils/mempoolTxUrl.ts

```diff
@@ -9,15 +9,15 @@ export default function mempoolTxUrl(txid?: string, network?: Network) {
     if (network) {
         switch (network) {
             case "bitcoin":
-                return `https://mempool.space/tx/${txid}`;
+                return `https://mutiny.mempool.space/tx/${txid}`;
```

Comment:
these are just for links, they redirect to `mempool.space` anyway... no big deal either way